.github/workflows/ci.yml

```diff
@@ -18,7 +18,12 @@ jobs:
       - name: Setup Babashka
         uses: turtlequeue/setup-babashka@v1.3.0
         with:
-          babashka-version: 0.7.0
+          babashka-version: 0.7.3
+
+      - name: Register bb clojure as clojure command
+        run: |
+          echo -e '#!/usr/bin/env bash\n\nbb --clojure "$@"' > /usr/local/bin/clojure
+          chmod +x /usr/local/bin/clojure
```

Comment:
This seems like it's about `nbb --clojure`. I assume it doesn't belong here?

Reply:
This is necessary for shadow-cljs which invokes `clojure` for invoking itself. But we already have bb installed which has a clojure runner: `bb clojure`. The n is a newline.

Reply:
Ah! Misread.